src/utility.ts

```diff
@@ -27,6 +28,56 @@ class Utility {
   private static strBreak =
     "************************************************************";
 
+  /**
+   * Checks to see if 'xdg-open' is on the path
+   */
+  isXdgOpenAvailable(): boolean {
+    if (this.isFileOnPath("xdg-open")) {
+      return true;
+    }
+    return false;
+  }
+
+  /**
+   * For each platform, create a way to open a file with the filetypes
+   * default application.
+   *
+   */
+  openWithDefaultApp(file: string) {
+    switch (os.platform()) {
+      case "win32": {
+        spawnSync(`start ${file}`);
+        break;
+      }
+      case "darwin": {
+        spawnSync(`open ${file}`);
+        break;
+      }
+      case "linux": {
+        if (this.isXdgOpenAvailable()) {
+          spawnSync("xdg-open");
```

Comment:
still need to pass the file as an argument right?

Reply:
Yes. I have the flu and that was the moment things went from bad to worse.

Reply:
hope you feel better soon. thanks for fixing this